environments/operator/obs-operator.jsonnet

```diff
@@ -1,3 +1,18 @@
-(import '../../components/observatorium.libsonnet') + {
-  config+:: (import 'operator-config.libsonnet'),
-} + (import '../../components/observatorium-configure.libsonnet')
+local config = import 'operator-config.libsonnet';
+local manifests = ((import '../../components/observatorium.libsonnet') + {
+                     config+:: config,
+                   } + (import '../../components/observatorium-configure.libsonnet')).manifests;
+
+local addOwner(k, v) = v { metadata+: { ownerReferences: [{
```

Comment:
we tend to put fields of nested objects on new lines, so something like:

```jsonnet
... v {
  metadata+: {
    ownerReferences: [{
      ...
    }],
  },
}
```

Reply:
Done